src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/boxes/AC8899.scala

```diff
@@ -28,7 +28,9 @@ case class AC8899(value: Option[Boolean]) extends CtBoxIdentifier(name = "Direct
   with DirectorsReportEnabledCalculator {
 
   override def validate(boxRetriever: Frs10xDirectorsBoxRetriever with FilingAttributesBoxValueRetriever): Set[CtValidation] =
-    failIf(boxRetriever.directorsReportEnabled().value) (
-      validateBooleanAsTrue("AC8899", this)
-    )
+    if (directorsReportEnabled(boxRetriever))
```

Comment:
We backed out the changes for dormancy that hadn't already been put into production (we don't have the qa resource to check the changes and there are several higher priority releases) - this change is just the result of that git revert. We'll revert the revert when we reintroduce these changes.